src/components/OTP.vue

```diff
@@ -384,6 +392,7 @@ export default {
       const responseStatusCode = responseStatusCodeAndMessage[1];
       if (responseStatusMessage.trim() === "success") {
         this.isLoading = true;
+        this.$store.dispatch("setPhoneNumber", this.phoneNumber);
```

Comment:
Use `mapActions` for this which is another helper like mapState. you can take a look at plio's code to get an idea how to use this